arm/ripper/makemkv.py

```diff
@@ -1074,14 +1075,18 @@ def run(options, select):
         raise TypeError(options)
     if not isinstance(select, OutputType):
         raise TypeError(select)
+    # Check makemkvcon path, resolves baremetal unique install issues
+    # Docker container uses /usr/local/bin/makemkvcon
+    makemkvcon_path = shutil.which("makemkvcon") or "/usr/local/bin/makemkvcon"
```

Comment:
thanks for the fix. I just migrated to bare metal ubuntu and stubled on this, too.